src/test/java/elysia/storage/StorageTest.java

```diff
@@ -0,0 +1,103 @@
+package elysia.storage;
+
+import elysia.command.Deadline;
+import elysia.command.Event;
+import elysia.command.Task;
+import elysia.command.ToDos;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
```

Comment:
To add on to liauzhanyi, maybe to help remedy the issue of wildcard imports, and assuming you are using IntelliJ as your IDE, you can:

`Go to settings -> Editor -> Code style -> Java -> Imports`

And under the "Class count to use import with *" field, you can change the number to 10000 or any large number.
You can also change the "Names count to use static import with *" for static imports.
This will ensure that IntelliJ will not classify your imports from the same package into a wildcard import (Unless you have more than 10000 imports 🤣)